README.md

```diff
@@ -8,23 +8,30 @@ An [oEmbed](http://oembed.com/) provider for embedding resources from the Stanfo
 
 There is an embedded static page available at `/pages/sandbox` in your development and test environments. Make sure that you use the same host on the service input (first text field) as you are accessing the site from (e.g. localhost or 127.0.0.1).
 
-## oEmbed specification details
+To bring up a dev environment first you'll need to install Ruby and JavaScript dependencies. Note: NodeJS v14 and [yarn](https://yarnpkg.com/) must be installed:
 
-URL scheme: `http://purl.stanford.edu/*`
+    bundle install
+    yarn install
 
-API endpoint: `TBD`
+Then start up the Rails app in one terminal window:
 
-Example: `TBD?url=http://purl.stanford.edu/zw200wd8767&format=json`
+    bundle exec bin/rails server
+
+Then start the NodeJS process to make the JavaScript assets available:
+
+    bin/webpack-dev-server
 
-## Installing JavaScript dependencies using Yarn
+Now visit this URL in your browser!
 
-Sul-Embed is starting to manage its JavaScript dependencies using [Yarn](https://yarnpkg.com/en/docs/install).
+    http://localhost:3000/pages/sandbox
 
-To install needed JavaScript dependencies make sure to install them using:
+## oEmbed specification details
+
+URL scheme: `http://purl.stanford.edu/*`
 
-```sh
-$ yarn install
-```
+API endpoint: `TBD`
```

Comment:
what's it mean that this is TBD?

Reply:
...I guess that was already there. I think this should be `embed.stanford.edu` now, probably?

Reply:
I guess?

Reply:
Either that or we can consider renaming the prod embed service `TBD.stanford.edu`!

Reply:
This was out of the scope of the changes I was making to just make it a bit easier to get up and running for development. But I just pushed a change to make this hostname and the example to hopefully reflect reality.